task2./script.js

```diff
@@ -0,0 +1,8 @@
+const button = document.getElementById("btn__element");
```

Comment:
Stenkitės galutinį rezultatą palikti švarų ir tvarkingą. Kelios to paties sprendimo versijos gali būtų jums nenaudingos, nes dėstytojas gali netyčia imti vertinti "prastesnę" versiją. :)